package.json

```diff
@@ -61,16 +58,24 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-import-resolver-babel-module": "^5.3.1",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.30.1",
     "gitbook-cli": "^2.3.2",
-    "metro-react-native-babel-preset": "^0.67.0",
-    "prettier": "^2.6.2",
-    "prettier-eslint": "^15.0.0",
-    "react": "17.0.2",
-    "react-native": "0.68.2",
-    "react-native-windows": "0.68.3"
+    "metro-react-native-babel-preset": "^0.72.0",
+    "prettier": "^2.7.1",
+    "prettier-eslint": "^15.0.1",
+    "react": "18.1.0",
+    "react-native": "0.70.0-rc.2",
```

Comment:
I believe it would be better to support RN 0.68+ instead of jumping all the way to RN 0.70 (since it's not out yet)

Reply:
There are so many improvements to cli and codegen and bug fixes. Although this is just fabric example app, we need to keep it updated as much as we can to make sure everything is well aligned with the new architecture.

Reply:
It would be awesome if you could at least try to support RN 0.69+, since otherwise adoption for this will be slow (e.g. folks will need to be on the latest version of RN, which isn't always possible right away).

From my understanding 0.70 brings auto-linking on Android, but that can be achieved via [this workaround](https://github.com/th3rdwave/react-native-safe-area-context/blob/d14de35ac6faa1dd61d370e04443999a86f2d5b2/android/src/main/jni/SafeAreaContextComponentsRegistry.cpp#L53-L59), or you can tell folks to manually link the module in their app `ComponentsRegistry.cpp` when using RN 0.69.

Unless 0.70 brings in some bugfix that does block this migration, and I'd be curious to learn more 🙌

Reply:
Fair point.

Reply:
You will be able to use the lib with any RN version that supports the new arch. We want require a certain, just recommend one.

Reply:
@capezzbr FYI, for the initial version of fabric support for this lib, we will probably stick to only supporting 0.70.x and newer. If I happen to find some additional time in the future, I definitely will take care of adding support for 0.68.x and higher.